_includes/header.html

```diff
@@ -11,7 +11,7 @@
   style="--image: url('{{ image }}')"
   data-dark="{{ dark }}"
   {% if page.url == "/" %}
-    data-big
+    data
   {% endif %}
```

Comment:
Can you just remove all these lines? From my reading of the style file, the header is either "data-big" or "not data-big", so I think you can just omit this.

Reply:
Right! I will delete that!

Reply:
Thanks! You should also be able to remove all of that `data-big` styling from the SCSS file too. I don't imagine we're going to want to go back to a separate header for the home page, and we can always roll back if we need it.

Reply:
By which I mean, remove the specialization of the header for `data-big`, and you should also then be able to remove the condition on `not data-big`.